data_store/validation/towns_fund/fund_specific_validation/resources/TF-grant-awarded.csv

```diff
@@ -810,3 +816,4 @@ TD-WRK-02,700000,7500000,8200000
 TD-WRK-03,0,4000000,4000000
 TD-WRK-04,0,5400000,5400000
 TD-WRK-05,0,1000000,1000000
+,,,
```

Comment:
As done this one

Reply:
Due to the same mistake as the other one, not going back to the beginning of the line in the terminal when copying data in. Now it's fixed.